### PR TITLE
fix(protractor): update specs for protractor 1.6

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: tasks/options/template.js [hunk 1]
```diff
@@ -29,11 +29,10 @@ module.exports = function(grunt) {
     }
 
     var files = {'<%= distDir %>/index.html': '<%= appDir %>/index.html'};
-    var APIARY_URL = process.env.APIARY_URL || 'http://superdesk.apiary.io';
 
     return {
         mock: {
-            options: data(APIARY_URL, true),
+            options: data('http://localhost:5000/api', true),
```
Comment (amagdas) [inline tasks/options/template.js]: :100:  :heart: :blue_heart: :yellow_heart:

**Comment by amagdas** [comment]
Does this fix https://dev.sourcefabric.org/browse/SD-1519 as well?

**Comment by petrjasek** [comment]
not sure, but you can assign it to me

**Comment by amagdas** [comment]
:+1:

**Pull request merged**